packages/api/src/graphql/comment-rating/comment-rating.public-mutation.ts

```diff
@@ -7,6 +7,16 @@ import {
 } from '../../error'
 import {SettingName} from '../../db/setting'
 
+export const validateCommentRatingValue = (type: RatingSystemType, value: number) => {
```

Comment:
Extracted so it can be used in the overriding part too

Reply:
like it!